yodeploy/virtualenv.py

```diff
@@ -164,6 +151,9 @@ def relocateable_ve(ve_dir):
     log.debug('Making virtualenv relocatable')
     virtualenv.make_environment_relocatable(ve_dir)
 
+    fix_local_symlinks(ve_dir)
```

Comment:
Although, I think one actually wants a mix. You want relative within the virtualenv, and absolute when referring to the parent python.